### PR TITLE
[zh] Sync setup/additional-setup/cni/

### PR DESCRIPTION
**Pull request by windsonsea**
Update a SVG and a phrase `init container` to keep consistent with [k8s init container](https://kubernetes.io/zh-cn/docs/concepts/workloads/pods/init-containers/), and add some new zh text

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure
- [x] Localization/Translation

See [preview](https://deploy-preview-14467--preliminary-istio.netlify.app/latest/zh/docs/setup/additional-setup/cni/#install-istio-with-cni-plugin) please.

### CODE REVIEW

File: content/zh/docs/setup/additional-setup/cni/index.md [hunk 1]
```diff
@@ -171,7 +171,7 @@ CNI 组件可以使用一个 `IstioOperator` 资源与控制平面一起升级
 apiVersion: install.istio.io/v1alpha1
 kind: IstioOperator
 spec:
-  profile: empty # Do not include other components
+  profile: empty # 不要包含其他组件
```
Comment (hanxiaop) [inline content/zh/docs/setup/additional-setup/cni/index.md:174]:
```suggestion
  profile: empty # 不包含其他组件
```

Comment (hanxiaop) [reply content/zh/docs/setup/additional-setup/cni/index.md:174]: 我觉得其实这个中文直接翻译出来和英文意思不太一样，empty profile 其实是不包含任何组件:
https://github.com/istio/istio/blob/master/manifests/profiles/empty.yaml#L1-L2
可以把这行注释删掉，在下面 cni: enabled: true 后面加上 # 只包含 CNI 组件

Comment (windsonsea) [reply content/zh/docs/setup/additional-setup/cni/index.md:174]: 额，注释掉的话，那这句也不用翻了吧，类似这样：

```
  # profile: empty # Do not include other components
```

Comment (hanxiaop) [reply content/zh/docs/setup/additional-setup/cni/index.md:174]: `profile: empty` 这行不能注释掉，是必须的配置。要不就还是翻译成 `不包含其他组件` 吧

Comment (windsonsea) [reply content/zh/docs/setup/additional-setup/cni/index.md:174]: ok

Comment (windsonsea) [reply content/zh/docs/setup/additional-setup/cni/index.md]: updated

**Review by hanxiaop — APPROVED** [review]

**Pull request merged**